oaknational/Oak-Web-Application
# Fix/homepage form a11y

## Description

- adds invisible labels to `<Input />`
- makes `label` a mandatory prop for `<Input />`
- corrects the SiteFooter tag (h4 -> h2) because axe was complaining
- changes pagination aria labels to be "next page" / "previous page" and adds nav aria label of "pagination" in line with recommendations here https://a11y-style-guide.com/style-guide/section-navigation.html

## Issue(s)

Fixes #578 

## How to test

1. Go to {cloud link}
2. Use Wave
3. You should see only 1 missing label error (this last one is in the lib we use so will raise a ticket on there https://github.com/oaknational/Oak-Web-Application/issues/590)
4. you should no longer see the heading order error

## Screenshots

![image](https://user-images.githubusercontent.com/12934669/188112991-1481fb88-a546-48f8-972b-5ed06a13f56a.png)

## Checklist
